packages/app/src/specs/SpecsListHeader.cy.tsx

```diff
@@ -4,27 +4,47 @@ import { defaultMessages } from '@cy/i18n'
 
 const buttonSelector = '[data-cy=new-spec-button]'
 
+const mountWithSearchRef = () => {
+  const search = ref('')
+
+  cy.wrap(search).as('search')
+
+  cy.mount(defineComponent({
+    setup () {
+      return () => h(SpecsListHeader, {
+        modelValue: search.value,
+        'onUpdate:modelValue': (val: string) => {
+          search.value = val
+        },
+      })
+    },
+  }))
+}
```

Comment:
How come you're using a search ref as opposed to just `cy.get`? Was it not working as expected?

Reply:
Ah maybe because you need the `modelValue` prop?

Reply:
Altho I'd think `modelValue` would just update on its own in accordance with the component behavior... haha sorry stream of consciousness over here